packages/ui/src/components/avatar/avatar.web.stories.tsx

```diff
@@ -27,7 +26,7 @@ export const Avatar: Story = {
   args: {
     children: (
       <>
-        <Component.Image alt="ST" src={StampsAvatarIconSrc} />
+        <Component.Image alt="ST" src="" />
```

Comment:
I'm changing this as we decided against including assets in the mono-repo: https://github.com/leather-io/issues/issues/103 

Loading the image will work via `src` once passed